scan

```diff
@@ -359,12 +372,16 @@ def perform_scan(params):
                 logging.warn("\t%s" % error)
 
         # If --meta wasn't requested, throw it all away.
-        if not options.get("meta", False): meta = None
+        if not options.get("meta", False):
+            meta = None
 
-        runner.write_rows(rows, domain, utils.base_domain_for(domain), scanner, handles[scanner]['writer'], meta=meta)
+        runner.write_rows(rows, domain,
+                          utils.base_domain_for(domain, cache_dir=cache_dir),
+                          scanner, handles[scanner]['writer'], meta=meta)
```

Comment:
If this one's going to be broken up over multiple lines (and I can see the value from a readability perspective), it's more readable with indentation like this:

```python
runner.write_rows(
    rows, domain,
    utils.base_domain_for(domain, cache_dir=cache_dir),
    scanner, handles[scanner]['writer'], meta=meta
)
```

I could see breaking it up into more lines than that, but the important thing is not to base the indentation of the arguments on the length of the name of the method call being made.

Reply:
I don't have a strong preference between these two approaches and am happy to change it.